src/matches.ts

```diff
@@ -13,7 +35,12 @@ function fuzzyMatches(textToMatch, node, matcher, normalizer) {
   }
 }
 
-function matches(textToMatch, node, matcher, normalizer) {
+function matches(
+  textToMatch: string | undefined,
```

Comment:
Can you have an undefined positional argument?

Reply:
`matches(undefined, node, matcher, normalizer)`?